src/io/read/mod.rs

```diff
@@ -485,7 +485,7 @@ mod tests {
     #[test]
     fn test_read_by_ref() {
         crate::task::block_on(async {
-            let mut f = io::Cursor::new(vec![0u8, 1, 2, 3, 4, 5, 6, 7, 8]);
+            let mut f = io::Cursor::new(vec![0_u8, 1, 2, 3, 4, 5, 6, 7, 8]);
```

Comment:
This seems like churn...

Reply:
Just more readable, and idiomatic/conventional to have an underscore in Rust. I agree that its not strictly necessary at all